sympy/printing/tests/test_latex.py

```diff
@@ -504,6 +504,15 @@ def test_latex_union():
         r"\left\{1, 2\right\} \cup \left[3, 4\right]"
 
 
+def test_latex_symmetric_difference():
+    assert latex(SymmetricDifference(Interval(2, 5), Interval(3, 5))) == \
```

Comment:
Only the test with `evaluate=False` should be enough. You should remove other two tests.